.travis.yml

```diff
@@ -7,9 +7,8 @@ cache:
   directories:
     - ~/.m2/repository
 install:
-  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -P travis
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true --quiet --batch-mode --show-version --activate-profiles travis
 before_script:
   - echo "MAVEN_OPTS=${TEST_MEM_SETTINGS}" > ~/.mavenrc
 script:
-  - mvn clean test cobertura:cobertura coveralls:report -B -P travis -DrepoToken=${COVERALLS_REPO_TOKEN} -Dtest.arguments="${TEST_MEM_SETTINGS}"
-
+  - mvn cobertura:cobertura coveralls:report --batch-mode --activate-profiles travis -DrepoToken=${COVERALLS_REPO_TOKEN} -Dtest.arguments="${TEST_MEM_SETTINGS}"
```

Comment:
still need to set the active profile here since you removed it from the pom? same in the 'install' line above

Reply:
yes, it's still need it... the profile in this pom was only for debugging

Reply:
The "travis" profile is defined in the parent pom so we still need that in order to kick in Coveralls, the bit I removed from this project's pom was an extension of that profile to not log to the console which we no longer need.